scapy/contrib/bgp.py

```diff
@@ -2004,6 +2006,31 @@ def post_build(self, p, pay):
         return p + pay
 
 
+#
+# LARGE_COMMUNITY
+#
+
+class BGPPALargeCommunity(Packet):
+    """
+    Provides an implementation of the LARGE_COMMUNITY attribute.
+    References: RFC 8092, RFC 8195
+    """
+
+    class LargeCommunitySegment(Packet):
```

Comment:
Could you rename this `BGPLargeCommunitySegment` and move it out of the class?

Reply:
Moved and renamed. I admit I was copying the style of BGPPAASPath class, but now I notice there is another style in "bgp.py".